cpp/benchmarks/CMakeLists.txt

```diff
@@ -183,6 +183,11 @@ ConfigureNVBench(
   sort/sort_lists.cpp sort/sort_structs.cpp
 )
 
+# ##################################################################################################
+# * create benchmark
+# --------------------------------------------------------------------------------
+ConfigureNVBench(CREATE_NVBENCH create/create_structs.cpp)
```

Comment:
Let's call something else like `STRUCT_CREATION_NVBENCH`. The prefix `CREATE_` is not clear.

Reply:
Also please post the comparison of benchmark results. FYI, the comparison is done by
```
python path_to_nvbench/nvbench_compare.py ./before.json after.json
```
where the json files generated by
```
./BENCH_NAME_NVBENCH --json file.json
```

Reply:
Thanks @ttnghia, I have updated the PR.